src/SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests/Features/ChangeApprenticeshipSteps.cs

```diff
@@ -122,34 +152,79 @@ public void ThenTheCommitmentStatementExistsInDatabase()
             });
         }
 
+        [Then(@"we have updated the apprenticeship details for the unconfirmed registration")]
+        public void ThenWeHaveUpdatedTheApprenticeshipDetailsForTheUnconfirmedRegistration()
+        {
+            _context.DbContext.Registrations.Should().ContainEquivalentOf(new 
+            {
+                Apprenticeship = new
```

Comment:
Do you think it might be worth including the original row's `ApprenticeId` here to prove we are *updating* a record and not just adding a new one?

Reply:
will do